scripts/install/install_macos.sh

```diff
@@ -360,106 +415,104 @@ set_opamp_secret_key()
   fi
 }
 
-set_formula_name()
+# latest_version gets the tag of the latest release.
+# It's also super hacky; It assumes that the output is formatted prettily.
+# Ideally we'd use jq, but we want to avoid external dependencies
```

Comment:
Can we remove these two lines? It would be fine if this was a dev comment in source but this is a customer facing script that they can easily read.